skypy/galaxies/stellar_mass.py

```diff
@@ -74,3 +78,131 @@ def schechter_smf_mass(redshift, alpha, m_star, m_min, m_max, size=None,
+        Amplitude of the Schechter mass function.
+
+     References
+    ----------
+    .. [1] de la Bella et al. 2021, Quenching and Galaxy Demographics,
+           arXiv 2112.11110.
+
+    '''
+
+    if np.ndim(phi_blue_total) == 1 and np.ndim(fsatellite) == 1:
+        phi_blue_total = phi_blue_total[:, np.newaxis]
+
+    sum_phics = (1 - fsatellite) * (1 - np.log(1 - fsatellite))
+
+    return (1 - fsatellite) * phi_blue_total / sum_phics
+
+
+def schechter_smf_amplitude_satellites(phi_centrals, fsatellite):
+    r'''Schechter amplitude of satellites.
+    This function returns the Schechter mass function amplitude
+    for active satellite population based on equation (15)
+    in de la Bella et al. 2021 [1]_.
+
+    Parameters
+    ----------
+    phi_centrals: float, (nt, nm) array_like
+        Schechter mass function amplitude of the central
+        active galaxies.
+
+    fsatellite: float, (nm, ) array_like
+        Fraction of active satellite galaxies between 0 and 1.
+        It could be a float or an array, depending on the model you choose.
+
+    Returns
+    -------
+    amplitude: float, (nt, nm) array_like
+        Amplitude of the Schechter mass function.
+
+    References
+    ----------
+    .. [1] de la Bella et al. 2021, Quenching and Galaxy Demographics,
+           arXiv 2112.11110.
+
+    '''
+
+    return phi_centrals * np.log(1 / (1 - fsatellite))
+
+
+def schechter_smf_amplitude_mass_quenched(phi_centrals, phi_satellites):
+    r'''Schechter amplitude of satellites.
+    This function returns the Schechter mass function amplitude
+    for passive mass-quenched population based on equation (15)
+    in de la Bella et al. 2021 [1]_.
+
+    Parameters
+    ----------
+    phi_centrals: float, (nt, nm) array_like
+        Schechter mass function amplitude of the central
+        active galaxies.
+
+    phi_satellites: float, (nt, nm) array_like
+        Schechter mass function amplitude of the satellite
+        active galaxies.
+
+    Returns
+    -------
+    amplitude: float, (nt, nm) array_like
+        Amplitude of the Schechter mass function.
+
+    References
+    ----------
+    .. [1] de la Bella et al. 2021, Quenching and Galaxy Demographics,
+           arXiv 2112.11110.
+
+    '''
+
+    return phi_centrals + phi_satellites
+
+
+def schechter_smf_amplitude_satellite_quenched(phi_satellites, fenvironment):
+    r'''Schechter amplitude of satellites.
+    This function returns the Schechter mass function amplitude
+    for active central population based on equation (15)
+    in de la Bella et al. 2021 [1]_.
+
+    Parameters
+    ----------
+    phi_satellites: float, (nt, nm) array_like
+        Schechter mass function amplitude of the satellite
+        active galaxies.
+
+    fenvironment: float
+        Fraction of satellite-quenched galaxies between 0 and 1.
+
+    Returns
+    -------
+    amplitude: float, (nt, nm) array_like
+        Amplitude of the Schechter mass function.
```

Comment:
```suggestion
    phi_satellites: array_like
        Schechter mass function amplitude of the satellite
        active galaxies.

    fenvironment: array_like
        Fraction of satellite-quenched galaxies between 0 and 1.

    Returns
    -------
    amplitude: array_like
        Amplitude of the Schechter mass function.
```
Surely `fenvironment` could also be an array here?

Reply:
Could be, `fenvironment` could depend on time, although not mass.